### PR TITLE
[BISERVER-5014] As a system admin, I want the User, IP address and th…

### PR DESCRIPTION
**Pull request by bennychow**
…e generated MDX/SQL for all reports executed on the Pentaho BA server.

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 1m 46s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 25s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **597**, Failures: **0**, Skipped: **3** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/platform-reporting-plugin/job/PR-763/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by bennychow** [comment]
@dcleao Hi Duarte, could you help merge this one?  After this one is merged, then I can make sure PIR builds because PIR depends on this reporting plugin.  Thanks.

**Review by dcleao — APPROVED** [review]

**Pull request merged**
